src/prototype/dom/layout.js

```diff
@@ -1677,27 +1530,6 @@
      !Element.descendantOf(element, document.body);
   }
 
-  // If the browser supports the nonstandard `getBoundingClientRect`
-  // (currently only IE and Firefox), it becomes far easier to obtain
-  // true offsets.
-  if ('getBoundingClientRect' in document.documentElement) {
```

Comment:
Does this really need to go? If anything I figure more browsers are supporting `getBoundingClientRect` these days.